src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Export/ExportJobTaskTests.cs

```diff
@@ -502,6 +507,75 @@ public async Task GivenConnectingToDestinationFails_WhenExecuted_ThenJobStatusSh
             Assert.Equal(HttpStatusCode.BadRequest, _lastExportJobOutcome.JobRecord.FailureDetails.FailureStatusCode);
         }
 
+        [Fact]
+        public async Task GivenStorageAccountConnectionChanged_WhenExecuted_ThenJobStatusShouldBeUpdatedToFailed()
```

Comment:
nit: Might be good to add one more test which validates that given `storageAccountConnection` has a value and remains same, then export succeeds as expected.